CBlue/README.md

```diff
@@ -53,6 +53,14 @@ If the module is not present,
 * Then restart X (control+alt+backspace).
 
 
+## Plank's theme
+
+* Copy the file `~/.theme/CBlue/cinnamon/dock.theme` to `~/.local/share/plank/themes/CBlue/`.
+* I recommend the following settings:
+
+![](https://pix.milkywan.fr/JPwBw1q0)
```

Comment:
You can upload images to GitHub and use the static URL that is generated. The files will exist as long as your account does. (You can upload to any dialog box to get the generated URL as a quick method.)

Reply:
These two links are permanent.
I don't know how to upload with github. Can you give me a link?
And do I have to make a new commit?

Reply:
Apologies since I'm not familiar with the domain. As an example, you can attach the files to the dialog box you used to write the comment. That will upload the files to GitHub servers and provide you the URL. If it's not too much, you can take those GitHub URLs and push one more commit to your existing branch and it will update this existing pull request.

GitHub's documentation is [here](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository).